numba/cpython/builtins.py

```diff
@@ -401,13 +403,11 @@ def next_impl(context, builder, sig, args):
 
 # -----------------------------------------------------------------------------
 
-@lower_builtin("not in", types.Any, types.Any)
-def not_in(context, builder, sig, args):
-    def in_impl(a, b):
+@overload("not in")
+def impl_not_in(a, b):
+    def impl(a, b):
         return operator.contains(b, a)
-
-    res = context.compile_internal(builder, in_impl, sig, args)
-    return builder.not_(res)
+    return impl
```

Comment:
Not sure that this is correct.
1. String `"not in"` looks like a workaround for something that should now be done differently. Am a little surprised that `@overload` tolerates this.
2. The original implementation has a negated result which is not reflected in these changes, is that intentional?

Further, I'm not convinced that this code is even reachable any more. The interpreter directly handles the `CONTAINS` bytecode and inverts the condition based on the op case flag:
https://github.com/numba/numba/blob/deeeaa273c7b2645f83d14f092eb124a19bfe40c/numba/core/interpreter.py#L2584-L2591